pimd/pim_cmd.c

```diff
@@ -723,9 +723,9 @@ static void igmp_show_interfaces_single(struct pim_instance *pim,
 				vty_out(vty, "Interface : %s\n", ifp->name);
 				vty_out(vty, "State     : %s\n",
 					if_is_up(ifp)
-						? (igmp->mtrace_only ? "mtrace"
-								     : "up")
-						: "down");
+					? (igmp->mtrace_only ? "mtrace"
+					   : "up")
+					: "down");
```

Comment:
clang-format did this? looks like crap

Reply:
well, I don't love the nested conditional, but the alignment is to the opening paren of the block? I can improve it by hand...